### DISCUSSION TITLE
Is Primefaces Freya Theme running in Tomcat7 ?

### DISCUSSION THREAD

**Discussion by engegov** [primefaces/community#1659] [PrimeFaces Templates]
I bougth the primefaces freya theme and install by maven in my eclipse IDE, but I need run in tomcat 7 like a model. 

Is there any way for running this template in tomcat7 support?

Thanks in advance.

**Comment by melloware** [comment]
Well i think you should start with this tutorial: https://balusc.omnifaces.org/2020/04/jsf-23-tutorial-with-eclipse-maven.html

**Comment by tandraschko** [comment]
Templates are built to run on a plain servlet container like Tomcat and Jetty, so yes

everything else depends on your IDE
